plugins/GSdx/GSRenderer.cpp

```diff
@@ -274,7 +287,7 @@ bool GSRenderer::Merge(int field)
 
 		if(m_regs->SMODE2.INT && m_interlace > 0)
 		{
-			if (m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
+			if(m_interlace == 7 && m_regs->SMODE2.FFMD == 1) // Auto interlace enabled / Odd frame interlace setting
```

Comment:
it's a bool yes, if it's not (a pointer for example) when not. Less code isn't always better. Especially if it still takes same amount of lines.